### ISSUE TITLE
Create and release a certifier binary

### ISSUE THREAD

**Issue by LucasRoesler** [openfaas/certifier#54]
It would be really great to have versioned binaries that we can then use in CI systems instead of needing to clone and run this repo. Having a built binary means we also don't need to worry about the go environment the user has.

we can use

```sh
go test -c ./tests -o faas-certifier
```
and this will produce a binary called `faas-certifier` for the tests.

**Comment by alexellis** [comment]
Please feel free to add a .travis.yml file for this and to add to the Trello board for tracking.

**Comment by LucasRoesler** [comment]
@alexellis i don't think this is ready until we update the README with instructions on how to use the docker image, but i always get confused about the networking.  Perhaps you have a suggestion on the right way to run it.

For example, with docker swarm on your local machine, you can do 

```sh
docker run --network=func_functions openfaas/certifier:latest -gateway=http://func_gateway:8080
```

But this presents an issue when running with auth, because you can just mount the local faas-cli config into the container because the gateway host names will be different, specifically, this doesn't work

```sh
docker run -v $HOME/.openfaas:/home/default/.openfaas --network=func_functions openfaas/certifier:latest -gateway=http://func_gateway:8080
```
